Open-Source-Pundits/nutshell
# Ts testing

# Description

Shoot I did some stuff you know it dog!

But seriously, got the session id for POST working for events.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.



# Testing Instructions

1. `git fetch --all`
2. `git checkout ts-testing`
3. `git origin ts-testing`
4. Make sure a new session posts events to API wo null value



# Checklist:
